### ISSUE TITLE
If enablePodSecurityPolicies is set to false, ClusterRole fails validation

### ISSUE THREAD

**Issue by ttyS0** [hashicorp/consul-helm#173]
The default value of `enablePodSecurityPolicies` is **false**. That value is placed above the `rules:` section in [server-clusterrole.yaml](https://github.com/hashicorp/consul-helm/blob/master/templates/server-clusterrole.yaml), which causes a broken `ClusterRole` resource to be described because the `rules:` section is mandatory.

The chart kicks out with:
```
Error: validation failed: error validating "": error validating data: ValidationError(ClusterRole): missing required field "rules" in io.k8s.api.rbac.v1.ClusterRole
```
unless `enablePodSecurityPolicies` is set to **true**.

**Comment by EDGsheryl** [comment]
I have encountered the same situation. It seems like that helm-v2.14.0 caused this problem.

**Comment by adilyse** [comment]
Hi, thanks for reporting this issue! Since this appears to be related to the Helm version issue also reported in #178, I'm going to close so there isn't a duplicate. Please feel free to track the status of the issue there.

**Comment by dverbeek84** [comment]
both server-clusterrole.yaml and client-clusterrole.yaml have a if statement block warpping the rules. thay should wrap the whole ClusterRole block.

**Comment by diegopaludo** [comment]
Hi,
I have encountered the same situation. :(
I solved it by upgrading to version v2.14.1 of helm

**Comment by dignajar** [comment]
Perfect, thanks!

For users with Mac OS and you installed via Helm via Brew.
```
brew upgrade kubernetes-helm
helm init --service-account tiller --upgrade
```
